source/languages/en/riak/dev/data-modeling/data-types.md

```diff
@@ -1125,3 +1324,12 @@ bruce.as_json()
  */
 ```
 
+```javascript
+/*
+ * Since the User model object does not contain Riak Data Type
+ * information internally, Javascript can serialize the object
+ * directly
+ */
+JSON.stringify(joe);
+```
+
```

Comment:
Does this last space need to be here for the example to render? I didn't think it did, but I could be wrong.